storage/storage_rules_test.go

```diff
@@ -470,10 +470,10 @@ func TestDBStorageGetVotes(t *testing.T) {
 	mustWriteReport3Rules(t, mockStorage)
 
 	helpers.FailOnError(t, mockStorage.VoteOnRule(
-		testdata.ClusterName, testdata.Rule1ID, testdata.UserID, types.UserVoteLike,
+		testdata.ClusterName, testdata.Rule1ID, testdata.UserID, types.UserVoteLike, "",
 	))
 	helpers.FailOnError(t, mockStorage.VoteOnRule(
-		testdata.ClusterName, testdata.Rule2ID, testdata.UserID, types.UserVoteDislike,
+		testdata.ClusterName, testdata.Rule2ID, testdata.UserID, types.UserVoteDislike, "",
 	))
 
 	feedbacks, err := mockStorage.GetUserFeedbackOnRules(
```

Comment:
Maybe you can add some test with a feedback string longer than 250 to check that the appropriate status code is returned

Reply:
Added test for long message